caffe2/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_subdirectory(proto)
 
 add_subdirectory(contrib)
 add_subdirectory(core)
+add_subdirectory(core/nomnigraph)
```

Comment:
Is nomnigraph supposed to be an integral part of Caffe2? If not, consider adding `USE_NOMNIGRAPH` option in top-level CMakeLists

Reply:
it is supposed to be an integral part of caffe2